conftest_source.py

```diff
@@ -325,6 +325,40 @@ def pytest_addoption(parser):
 
             # Fails due to the async batch changes
             'transport/test_ipc.py::IPCMessagePubSubCase::test_multi_client_reading',
+
+            # Needs investigation. Setting them to xfail to have a "new green start" on March 12th
```

Comment:
I would suggest to also link the Gh issue here:

```suggestion
            # Needs investigation. Setting them to xfail to have a "new green start" on March 12th
            # https://github.com/SUSE/spacewalk/issues/14263
```